doc/source/serve/managing-java-deployments.md

```diff
@@ -62,7 +62,7 @@ By specifying the `numReplicas` parameter, you can change the number of deployme
 
 ### Resource Management (CPUs, GPUs)
 
-Through the `rayActorOptions` parameter, you can set the resources of deployment, such as using one GPU:
+Through the `rayActorOptions` parameter, you can reserve resources for each deployment replica, such as one GPU:
```

Comment:
Is this correct? Does the `rayActorOptions` parameter control resources for each replica (like the Python API), or is it for each deployment?

Reply:
each replica, exactly the same as python